source/faqs.rst

```diff
@@ -373,3 +367,208 @@ The ``--no-builds`` argument completely removes the build number from the
+compiler('fortran') }}``. These should go in the ``build`` section, and all
+other build dependencies should go in the ``host`` section.
+
+Anaconda provides platform-specific compilers that are automatically
+determined. The string ``{{ compiler('c') }}`` will resolve to ``gcc`` on
+Linux, but ``clang`` on macOS (osx-64).
+
+
+.. seealso::
+
+    - The `compiler tools <conda-build:compiler-tools>` section of the
+      conda docs has much more info.
+
+    - The default compiler options are defined by conda-build in the
+      `variants.DEFAULT_COMPILERS
+      <https://github.com/conda/conda-build/blob/master/conda_build/variants.py#L42>`_
+      variable.
+
+    - More details on "strong" and "weak" exports (using examples of
+      libpng and libgcc) can be found in the `export runtime
+      requirements <conda-build:run_exports>` conda documentation.
+
+
+.. _global-pinning:
+
+How does global pinning work?
+-----------------------------
+
+We can have conflicts when the version of a common library used when the
+package is originally *built* differs from the version when the package is
+*installed*. All packages intending to be installed into the same environment
+should be built using the same versions of common libraries so that they can
+co-exist. **Global pinning** is the idea of making sure all recipes use the
+same versions of common libraries.
+
+For example, many bioinformatics tools have ``zlib`` as a dependency.
+The version of ``zlib`` used when building the package should be the same as the
+version used when installing the package into a new environment. This implies
+that we need to specify the ``zlib`` version in one place and have *all recipes
+intended to coexist in the same environment* use that version.
+
+This is configured with special build config files. Since we rely heavily on
+the conda-forge channel, the bioconda build system installs the conda-forge
+`conda_build_config.yaml
+<https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml>`_
+into our build environment so that it can be used for building all recipes.
+This is then combined with the bioconda-specific
+`bioconda-Utils-conda_build_config.yaml
+<https://github.com/bioconda/bioconda-utils/blob/master/bioconda_utils/bioconda_utils-conda_build_config.yaml>`_.
+Note that in some cases the bioconda config may override some of the
+conda-forge configs. For example, historically, we did this when we wanted to
+support older Python versions.
+
+The idea here is to specify that any time a dependency (``zlib``, in our
+running example) is used as a build dependency, it should also be automatically
+be installed as a run dependency without having to explicitly add it as such in
+the recipe. This specification is done in the ``zlib`` recipe itself (which is
+hosted by conda-forge), so in general bioconda collaborators can just add
+``zlib`` as a build dependency.
```

Comment:
I find this confusing.
https://github.com/bioconda/bioconda-docs/pull/16/files#diff-5a476a141b5805f24f43cab6c3548f1dd1319013abd9035ef91062a6fcd9a35bR441-R442 says that `zlib` should be in the `requirements:host` list. 
Here the terminology says `build dependency` which sounds like `requirements:build` which is only for the compilers, as explained earlier.

Reply:
I agree. I think "host dependency" or "host requirement" would be more clear.

Also, I'd like to share my related open PR #18 to clarify how to handle requirements like `zlib` that use "run exports"

Reply:
@jdblischak agreed, and will get to #18 soon. I think this should be fixed over there. Any suggestions on what a better example would be?